boards/grand_central_m4/src/lib.rs

```diff
@@ -595,18 +609,22 @@ pub fn uart(
 
 #[cfg(feature = "usb")]
 /// Convenience function for setting up USB
-pub fn usb_allocator(
+pub fn usb_allocator<G: GclkId>(
     usb: pac::Usb,
-    clocks: &mut GenericClockController,
-    mclk: &mut pac::Mclk,
-    dm: impl Into<UsbDm>,
-    dp: impl Into<UsbDp>,
-) -> UsbBusAllocator<UsbBus> {
-    use pac::gclk::{genctrl::Srcselect, pchctrl::Genselect};
+    dfll: Enabled<Dfll, U1>,
+    gclk_token: GclkToken<G>,
+    pclk_token: PclkToken<Usb>,
+    ahb_clk: AhbClk<clock::types::Usb>,
+    apb_clk: ApbClk<clock::types::Usb>,
+    dm: impl AnyPin<Id = PA24>,
+    dp: impl AnyPin<Id = PA25>,
+) -> UsbBusAllocator<UsbBus<G>> {
+    // SWITCH DFLL INTO USB CLOCK RECOVERY MODE - THIS ALLOWS FOR MORE ACCURATE USB
```

Comment:
I'd suggest moving the comment about clock recovery mode in to the docstring for the method, so that it shows up in docs (or Rust Analyzer etc).  Also, I'm pretty sure it's required to use USBCRM if running USB with the DFLL clock source, so not sure "more accurate" is the right way to describe it.